superset-frontend/src/views/CRUD/alert/AlertList.tsx

```diff
@@ -182,27 +184,33 @@ function AlertList({
           row: {
             original: { last_state: lastState },
           },
-        }: any) => <AlertStatusIcon state={lastState} />,
+        }: any) => (
+          <AlertStatusIcon
+            state={lastState}
+            isReportEnabled={isReportEnabled}
+          />
+        ),
         accessor: 'last_state',
         size: 'xs',
         disableSortBy: true,
       },
-      {
-        accessor: 'name',
-        Header: t('Name'),
-      },
       {
         Cell: ({
           row: {
-            original: { recipients },
+            original: { last_eval_dttm: lastEvalDttm },
           },
-        }: any) =>
-          recipients.map((r: any) => (
-            <RecipientIcon key={r.id} type={r.type} />
-          )),
-        accessor: 'recipients',
-        Header: t('Notification Method'),
-        disableSortBy: true,
+        }: any) => {
+          return lastEvalDttm
+            ? moment.utc(lastEvalDttm).local().format(DATETIME_WITH_TIME_ZONE)
```

Comment:
Unrelated but we should update other lists where we use `moment` to properly handle timezones